torch/csrc/distributed/rpc/rref_context.h

```diff
@@ -206,6 +206,9 @@ class TORCH_API RRefContext {
 
   std::unordered_map<std::string, std::string> getDebugInfo();
 
+  // use the given Events to block current streams on corresponding devices
+  void blockCurrentStreams(const std::vector<c10::Event>& events) const;
```

Comment:
Extra nit: this could be static. In fact, we don't even need this to be a method on RRefContext at all: it could just be a helper function in an anonymous namespace directly inside `rref_impl.cpp`.